etc/kayobe/environments/aufn-ceph/seed-hypervisor.yml

```diff
@@ -11,4 +11,4 @@ seed_hypervisor_extra_network_interfaces:
   - "{{ external_net_names[0] }}"
 
 # Workaround change to cloud-user default login name on CentOS-Stream8
-seed_hypervisor_bootstrap_user: "{{ lookup('env', 'USER') }}"
+seed_hypervisor_bootstrap_user: "{{ 'cloud-user' if os_distribution == 'centos' else os_distribution }}"
```

Comment:
This one we might want to keep as USER? Does your seed hypervisor image have a rocky user?

Reply:
Oh yes, good catch. The Rocky image we use on SMS has cloud-user as the user